### PR TITLE
[BACKLOG-8366] Fixed references to modules specific to VizAPI 2.5

### PR DESCRIPTION
**Pull request by carlosrusso**
@pentaho/millenniumfalcon

### CODE REVIEW

File: package-res/resources/web/pentaho/visual/2.5/samples/calc/visualTypeProvider.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/*!
+
```
Comment (dcleao) [inline package-res/resources/web/pentaho/visual/2.5/samples/calc/visualTypeProvider.js]: This looks like a typo on your end

**Comment by dcleao** [comment]
👍 (apart from the typo)

**Comment by carlosrusso** [comment]
@dcleao @pentaho/millenniumfalcon All set!

**Comment by dcleao** [comment]
Reviewed the new commits. All looks good.

**Comment by carlosrusso** [comment]
Rebased to master following #773

**Pull request merged**